packages/clay-pagination/src/PaginationEllipsis.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as React from 'react';
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+
+export interface PaginationEllipsisProps {
+	items?: number[];
+	disabledPages?: number[];
+	hrefConstructor?: (page?: number) => string;
+	onPageChange?: (page?: number) => void;
+}
+
+const PaginationEllipsis: React.FunctionComponent<PaginationEllipsisProps> = ({
+	disabledPages = [],
+	hrefConstructor,
+	items,
+	onPageChange,
+}) => {
+	const [active, setActive] = React.useState(false);
+
+	return (
+		<ClayDropDown
+			active={active}
+			className="page-item"
+			containerElement="li"
+			onActiveChange={newVal => setActive(newVal)}
+			trigger={<ClayButton className="page-link">{'...'}</ClayButton>}
```

Comment:
hey @bryceosterhaus maybe we'll just have to use the `button` tag, we'll have problems with the `btn-*`class that are added by default to the ClayButton.

Reply:
The clay docs show you can use clay-button with pagination. Check out, https://clayui.com/docs/css-framework/satellites/paginations.html. I forgot to add the `btn-unstyled` though, I will update that.